test/installation/assets/puppeteer/puppeteer-in-browser.js

```diff
@@ -0,0 +1,35 @@
+import puppeteer from 'puppeteer';
+
+const browser = await puppeteer.launch({
+  headless: true,
+  args: ['--remote-allow-origins=*'],
+});
+
+try {
+  const port = process.argv[2];
+
+  console.log(port);
+
+  const page = await browser.newPage();
+
+  await page.goto(`http://localhost:${port}/index.html`);
+
+  await page.type('input', browser.wsEndpoint());
+
+  const result = new Promise(resolve => {
+    page.once('dialog', dialog => {
+      dialog.accept();
+      resolve(dialog.message());
+    });
+  });
+
+  await page.click('button');
```

Comment:
Let's use Promise.all here it's better practice.

Reply:
It's more of a pattern to use/show users: Set up listener and execute the command at the same time.
I acknowledge that this is correct as is it.